.github/workflows/cache-dependencies.yml

```diff
@@ -76,6 +76,7 @@ jobs:
         with: 
           repository: openmc-dev/openmc
           path: openmc
+          submodules: reursive
```

Comment:
I am fairly unexperienced with github workflows. What does this line do in the workflow?

Reply:
See the [README](https://github.com/actions/checkout#usage) for the ``checkout`` action. We set this setting to ``recursive`` to match the command in the [instructions to install OpenMC from source.](https://docs.openmc.org/en/latest/usersguide/install.html#obtaining-the-source)